src/redux/store.js

```diff
@@ -1,12 +1,15 @@
 import { configureStore } from '@reduxjs/toolkit';
 import logger from 'redux-logger';
-import missionsReducer from './redux/missions';
+import thunk from 'redux-thunk';
+import missionsReducer from './missions';
+import rocketsReducer from './rockets/rockets';
 
 const store = configureStore({
   reducer: {
     missions: missionsReducer,
+    rockets: rocketsReducer,
   },
-  middleware: [logger],
+  middleware: [logger, thunk],
```

Comment:
[OPTIONAL] : 
`thunk` is already included in `configureStore` method, so it is not mandatory to put it here.

Reply:
Without adding it in the middleware, I was getting this error https://stackoverflow.com/questions/46765896/react-redux-actions-must-be-plain-objects-use-custom-middleware-for-async-acti

Reply:
It is curious because for me it is working without `thunks`.

We can keep it as it will ensure the async functions will work for both.